docs/en/ingest-management/elastic-agent/configuration/elastic-agent-kubernetes-autodiscovery.asciidoc

```diff
@@ -150,50 +150,53 @@ This is the full list of supported hints:
 
 The package to use for monitoring.
 
+[float]
```

Comment:
Does it make sense to make them a bullet list?

Reply:
I think it's just fine in this format. Not sure how we could represent it better with ascidoc. Maybe @dedemorton could provide some suggestions here?

Reply:
@ChrsMark  IMO the separate sections are OK for now because our docs are pretty inconsistent wrt how we format reference content (we use sections, description lists, and tables in various places). I'm waiting for clear consistency guidelines before I make any big changes to formatting across the docs.  I don't think those guidelines will come until we move to the new system.

If you decide to change the format, I would use a [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/), not bulleted lists. You'll need to use continuation characters (+) to keep multi-paragraph descriptions together.

As an alternative, I've tried using tables, but there are too many formatting problems, especially for elements with long dotted names.